src/models/Observation.js

```diff
@@ -424,9 +420,9 @@ class Observation extends Realm.Object {
     try {
       response = await inatjs.observations.create( uploadParams, options );
     } catch ( uploadError ) {
-      const body = JSON.parse( await uploadError.response.text( ) );
-      console.error( "[ERROR] Failed to upload observation: ", JSON.stringify( body ) );
-      return body;
+      const errorText = await uploadError.response.text( );
+      uploadError.message = errorText;
+      throw uploadError;
```

Comment:
I think this might be the reason uploads are no longer working for me. I'm getting an error thrown: `Error: {"status":"401","errors":[{"errorCode":"401","message":"JWT is missing"}]}`

Reply:
This was intentional. I'm not a huge fan of catching all errors all the time. Some errors really should never happen, so if they do, I think they should bubble up and not be caught, so we learn about them. In this case, it's not this different error handling that's causing the problem, it's that for some reason attempts are being made to upload an obs with a blank API token.